models/nlp/classification/huggingface/train.py

```diff
@@ -0,0 +1,84 @@
+import valohai
+from datasets import load_dataset
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    Trainer,
+    TrainerCallback,
+    TrainingArguments,
+)
+
+
+class PrintMetricsCallback(TrainerCallback):
+    def on_evaluate(self, args, state, control, metrics, **kwargs):
+        with valohai.logger() as logger:
+            for key in metrics:
+                logger.log(key, metrics[key])
```

Comment:
Assuming `metrics` is a dict,

```suggestion
            for key, value in metrics.items():
                logger.log(key, value)
```
which in turn means you can just
```suggestion
            logger.log(**metrics)
```